docs/sql-ref-syntax-dml-load.md

```diff
@@ -78,7 +78,7 @@ LOAD DATA [ LOCAL ] INPATH path [ OVERWRITE ] INTO TABLE table_identifier [ part
      | Amy Smith      | 123 Park Ave, San Jose         | 111111         |
      + -------------- + ------------------------------ + -------------- +
 
- CREATE TABLE test_load (name VARCHAR(64), address VARCHAR(64), student_id INT);
+ CREATE TABLE test_load (name VARCHAR(64), address VARCHAR(64), student_id INT) USING HIVE;
```

Comment:
This must be a hive table

Reply:
Hi, @cloud-fan . This reminds me.
```
SPARK-30098 Use default datasource as provider for CREATE TABLE syntax
```

@marmbrus and @gatorsmile . Do we need to revert SPARK-30098 due to its silent behavior change?

Also, cc @rxin since he is the release manager for 3.0.0.

Reply:
I filed SPARK-31136 to track the discussion and the final result.

Reply:
The cost of break is mostly users can't run `LOAD TABLE`?

Reply:
Please don't get me wrong. You know that I loved that patch. However, the new policy is designed to ban this kind of behavior change (SPARK-30098). Technically,
1. The benefit is just saving a few word `USING PARQUET` or something.
2. The downside is breaking the existing user pipelines.

Reply:
I updated the JIRA description with @cloud-fan 's example.

Reply:
> The benefit is just saving a few word USING PARQUET or something.

This is a completely underestimate. Please see my comment in https://github.com/apache/spark/pull/27894#issuecomment-598534428